go.mod

```diff
@@ -3,22 +3,20 @@ module github.com/ONSdigital/dp-image-api
 go 1.13
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.14.0
+	github.com/ONSdigital/dp-api-clients-go v1.15.0
 	github.com/ONSdigital/dp-authorisation v0.1.0
 	github.com/ONSdigital/dp-healthcheck v1.0.4
 	github.com/ONSdigital/dp-kafka v1.1.6
-	github.com/ONSdigital/dp-mongodb v1.2.0
+	github.com/ONSdigital/dp-mongodb v1.2.1-0.20200703105842-06d35548f9ba
```

Comment:
Is there a new release version we can use, or are we waiting on another PR?

Reply:
It's still in PR - I have just bumped it in the review channel, hopefully it can be released soon and then I will update this.